### PR TITLE
Save outputs of `Kube_job.{describe,logs}`

### PR DESCRIPTION
**Pull request by smondet**
This is a first step twoards fixing #37.

I'm still testing this, and then I'll the same for `logs`.

### CODE REVIEW

**Comment by smondet** [comment]
one Fresh:

screenshot 2016-09-01 18 10 37

one old:

screenshot 2016-09-02 15 41 28

File: src/ketrew_backend/plugin.ml [hunk 1]
```diff
@@ -229,8 +229,8 @@ module Long_running_implementation : Ketrew.Long_running.LONG_RUNNING = struct
           Client.get_kube_job_descriptions client [job_id]
           >>= fun l ->
           let rendered =
-            List.map l ~f:(fun (id, s) ->
-                sprintf "### Kube-Job %s\n\n%s" id  s)
+            List.map l ~f:(fun (`Id id, `Describe_output o, `Freshness frns) ->
+                sprintf "### Kube-Job %s\n### Freshness: %s\n\n\n%s" id frns o)
```
Comment (ihodes) [inline src/ketrew_backend/plugin.ml]: Freshness = status, which would be "running", or "ran" or something?

**Comment by ihodes** [comment]
LGTM 1 Q wrt "freshness"

File: src/ketrew_backend/plugin.ml (hunk 1 above)
Comment (smondet) [inline src/ketrew_backend/plugin.ml]: No, it's not the "status of the job" it's what kind of result is it returning:

- `fresh`: just got the results from `kubectl`
- `old err`: “`kubectl` failed with error `err`, and the result I'm showing the is the latest successful one I know about.”

**Comment by smondet** [comment]
@ihodes RE "Archived" the thing is that Kubectl can also fail randomly and then work the next time; so I like actually displaying the error, and just saying that the output is "old". Not sure I'm clear...

**Comment by ihodes** [comment]
That makes sense; still, could be Archived of boolean * string or something? I just find the named "archived" to better express what's going on (you're archiving the job in storage), but I don't feel very strongly about it.

**Pull request merged**